packages/bitcore-client/bin/wallet-register

```diff
@@ -10,12 +10,26 @@ program
   .option('--baseUrl <url>', 'optional - Change configured wallet baseUrl')
   .parse(process.argv);
 
+if (!program.path || !program.baseUrl) {
+  program.help();
+}
+
 const main = async () => {
   const { path, baseUrl } = program;
   let wallet;
   try {
     const password = await promptly.password('Wallet Password:');
+    if (!password) {
+      throw new Error(
+        'No password provided. Keys can\'t be imported without unlocking the wallet'
```

Comment:
I think the password can be empty, let's see.

Reply:
updated